tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -289,4 +336,16 @@ Object.Base {
 		source	"copier.SSP.6.1"
 		sink	"copier.host.5.1"
 	}
+
+	route."6" {
+		source	"copier.SSP.8.1"
+		sink	"copier.module.8.2"
```

Comment:
this route can go into the pipeline definition passthrough-copier-be.conf

Reply:
Let me think about it, thanks for your review!!